internal/util/test_env.go

```diff
@@ -0,0 +1,20 @@
+package util
```

Comment:
Having a helper used by all the controllers that resides in a file named `test_xxx` sounds weird to me; at first look, I expect that everything in such a file is used only by tests. What do you think of renaming this file?

Reply:
I agree - it's weird and it should stay like that. My intention was to not expose this env var as one of the official flags, thus 'hiding' it in this file.  

After some thought, I think that might be even a good thing to make it a regular flag: 
- It's always giving an opportunity to tweak the timeout in case somebody's setup cache sync takes longer than the default 2mins
- It doesn't introduce strangely hidden env vars which are meant to be used only in tests while they are still able to affect production runs
  
[2e7243a](https://github.com/Kong/kubernetes-ingress-controller/pull/3013/commits/2e7243ae264e85c8271bbc3127ec3cbda759c31d)